modules/server/src/main/java/edu/gemini/seqexec/server/tcs/ParkState.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+ * For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package edu.gemini.seqexec.server.tcs;
+
+public enum ParkState {
```

Comment:
Why are these in Java? Due to the XML parser?

Reply:
Due to the giapi epics library.